### ISSUE TITLE
MDAnalysis 1.0.0 tests fail with "TypeError: use() got an unexpected keyword argument 'warn'"

### ISSUE THREAD

**Issue by orbeckst** [MDAnalysis/mdanalysis#2938]
Note: **This problem (https://github.com/MDAnalysis/mdanalysis/issues/2191#issuecomment-664818527) was already fixed (in PR #2763) and will not affect upcoming 1.0.1**. This issue was raised so that current users of 1.0.0 have an easy way to find this problem that currently affects them. This issue will be closed when 1.0.1 is releases #2768 . 

## Expected behavior ##
Conda installed MDA "just works".

## Actual behavior ##
Following our installation instructions, MDAnalysis tests fail with 
```
miniconda3/envs/mdaenv/lib/python3.6/site-packages/MDAnalysisTests/__init__.py:126: in <module>
    matplotlib.use('agg', warn=False)
E   TypeError: use() got an unexpected keyword argument 'warn'
```

A new version of matplotlib (3.3.1 in my case) is installed that removed the `warn` kwarg.

## Code to reproduce the behavior ##

```
conda config --add channels conda-forge
conda update --yes conda
conda create --yes -n mdaenv python=3.6
conda install --yes -n mdaenv MDAnalysis MDAnalysisTests
source activate mdaenv
pytest --disable-pytest-warnings --pyargs MDAnalysisTests
```

## Workaround
Downgrade mpl
```
conda install 'matplotlib<3.3'
```

## Current version of MDAnalysis ##

- Which version are you using? (run `python -c "import MDAnalysis as mda; print(mda.__version__)"`) 1.0.0
- Which version of Python (`python -V`)? any
- Which operating system? any

**Comment by IAlibay** [comment]
This _should_ have been addressed in 1.0.1 (I'm running matplotlib-base 3.3.4 without any issues).
